docs/root/version_history/version_history.rst

```diff
@@ -7,6 +7,7 @@ Version history
   :titlesonly:
 
   current
+  v1.15.0
   v1.14.3
```

Comment:
```
1.12.6 
1.13.4
1.14.4
```
should be add here I guess

Reply:
This is `release/v1.15` branch, so I don't think so... we're not adding all those old releases here.

Reply:
(also, `v1.12.6`, `v1.13.4` and `v1.14.4` were released before `v1.15.0` was even cut)